content/pt-br/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -0,0 +1,160 @@
+
+## Exemplos
+
+Para ilustrar as diferentes maneiras de utilizar as *labels*, os exemplos abaixo apresentção uma complexidade variada. 
+
+### Um simples *Stateless Service*
+
+Considerando o case de um simples *stateless service* implantado utilizando os objetos `Deployment` e `Service`, os 2 trechos abaixo representam como as *labels* podem ser utilizadas da maneira mais simples possível.
+
+O `Deployment` é usado para supervisionar os *pods* que autoexecutam a aplicação.
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+    app.kubernetes.io/instance: myservice-abcxzy
+...
+```
+
+O `Service` usado para exibir a aplicação.
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+    app.kubernetes.io/instance: myservice-abcxzy
+...
+```
+
+### Aplicação Web com Banco de Dados
+
+Considerando uma aplicação razoavelmente complicada: uma aplicação web (WordPress) utilizando um banco de dados (MySQL), instalada com Helm. Os trechos a seguir mostra como iniciar os objetos usados para implantar a aplicação.
+
+Para iniciar exibindo como o `Deployment` é usado para o WordPress:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/instance: wordpress-abcxzy
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: server
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+O `Service` usado para exibir o WordPress:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/instance: wordpress-abcxzy
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: server
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+O MySQL é exposto como um `StatefulSet` com metadados para ele e para o aplicativo maior ao qual ele pertence: 
+
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: mysql
+    app.kubernetes.io/instance: mysql-abcxzy
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+O `Service` usado para exibir o MySQL como parte do WordPress:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: mysql
+    app.kubernetes.io/instance: mysql-abcxzy
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+Utilizando os comandos MySQL, `StatefulSet` e `Service`, você receberá notificação sobre ambos, MySQL e WordPress, o aplicativo maior está incluso.
```

Comment:
Some suggestions:
- "Utilizando os comandos MySQL, `StatefulSet` e `Service`" --> "No `StatefulSet` e no `Service` do MySQL"
- "você receberá notificação" --> "você observará que informações"

I think "o aplicativo maior" refers to WordPress in this context, so it could have a comma right afterwards.